.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Instruct the GitHub linguist tool to interpret the generated JS code as just
```

Comment:
Solid move, now I think only those doing development need to make a RAM reservation to compile `protoc` themselves...